lib/travis/build/addons/mariadb.rb

```diff
@@ -0,0 +1,33 @@
+require 'shellwords'
+require 'travis/build/addons/base'
+
+module Travis
+  module Build
+    class Addons
+      class Mariadb < Base
+        SUPER_USER_SAFE = true
+
+        MARIADB_GPG_KEY = '0xcbcb082a1bb943db'
+        MARIADB_MIRROR  = 'nyc2.mirrors.digitalocean.com'
```

Comment:
Out of curiosity I did a quick and dirty benchmark: https://travis-ci.org/youtube/vitess/builds/72222979#L668

nyc2 from Digital Ocean is the fastest mirror :-) Up to 50 MB/s compared to 3.5MB/s from James Madison University :)